packages/web3-eth-contract/src/types.ts

```diff
@@ -319,10 +319,10 @@ export interface NonPayableMethodObject<Inputs = unknown[], Outputs = unknown[]>
 	 *   ],
 	 *   "gasUsed": "0x7671"
 	 * }
-	 *  ```
+	 * ```
 	 */
 	createAccessList(
-		tx?: NonPayableCallOptions,
+		tx?: Omit<NonPayableCallOptions, 'nonce' | 'type'>,
```

Comment:
`nonce` and `type` allowed for `createAccessList`
<img width="401" alt="image" src="https://user-images.githubusercontent.com/20375610/233438214-3fc98c77-ec2d-41c4-880a-3d98a0ae46c9.png">